### ISSUE TITLE
How does the ULP read the wall clock

### ISSUE THREAD

**Issue by kumkee** [victor-chew/espclock4#1]
Hi Victor, 
This is not to raise any problem but rather to understand how it works. A few questions here.
1. Does it work for simple analogue clocks?
2. How does the ULP read the current shown time from the wall clock?
3. Do you have any reference documentation on the reading and calibration of analogue wall clocks?

**Comment by Mechotronic** [comment]
I don't think it does. I believe you're supposed to manually set the clock and then turn it on. This looks like it bypasses the entire circuit and drives the coil directly.
